frontend/packages/client/src/pages/BrowseCommunities.js

```diff
@@ -41,16 +41,12 @@ export default function BrowseCommunities() {
 
```

Comment:
we need this one to be updated (Line 29):
```
const { search, filters: filtersUrl } = useQueryParams({
    search: 'search',
    filters: 'filters',
  });
  const [searchText, setSearchText] = useState(search ? search : '');
  ```
  to be 
```
const { text, filters: filtersUrl } = useQueryParams({
    text: 'text',
    filters: 'filters',
  });
  const [searchText, setSearchText] = useState(text ? text : '');
  
  ```
  This will enable copy paste search url and you'll get the same search results and also navigating back from a community